libkbfs/md_journal_test.go

```diff
@@ -162,6 +170,36 @@ func checkIBRMDRange(t *testing.T, uid keybase1.UID,
 	}
 }
 
+func BenchmarkMDJournalBasicMDv2(b *testing.B) {
+	_, _, id, signer, ekg, bsplit, tempdir, j :=
+		setupMDJournalTest(b)
```

Comment:
You should pass an explicit MD version here, because eventually we will make v3 the default.